src/llv8-inl.h

```diff
@@ -352,13 +352,29 @@ inline Value DescriptorArray::GetValue(int index, Error& err) {
 }
 
 inline bool DescriptorArray::IsFieldDetails(Smi details) {
-  return (details.GetValue() & v8()->descriptor_array()->kPropertyTypeMask) ==
-         v8()->descriptor_array()->kFieldType;
+  constants::DescriptorArray* d = v8()->descriptor_array();
+  int64_t v = details.GetValue();
+
```

Comment:
I tried fairly verbose names and it didn't look too bad (IMHO!), maybe mine were even longer and tipped the balance back.
```
  return (value & descriptor_array->kPropertyAttributesMask) ==
         (descriptor_array->kPropertyAttributesEnum_READ_ONLY
          << descriptor_array->kPropertyAttributesShift);
```